src/test/java/net/wasdev/wlp/common/plugins/util/DevUtilTest.java

```diff
@@ -76,12 +85,150 @@ public void testCleanupServerEnvBak() throws Exception {
+        DevUtil util = new DevTestUtil(null, null, null, null, null, false);
+
+        File serverDirectory = Files.createTempDirectory("serverDirectory").toFile();
+        File tempFile = new File(serverDirectory, "temp.txt");
+        Files.write(tempFile.toPath(), "temp".getBytes());
+        String fileString = util.readFile(tempFile);
+
+        assertTrue(fileString.equals("temp"));
+
+        if (tempFile.exists()) {
+            tempFile.delete();
+        }
+    }
+
+    @Test
+    public void testCopyConfigFolder() throws Exception {
+    }
+
+    @Test
+    public void testCopyFile() throws Exception {
+        DevUtil util = new DevTestUtil(null, null, null, null, null, false);
+
+        File srcDir = Files.createTempDirectory("src").toFile();
+        File targetDir = Files.createTempDirectory("target").toFile();
+        File configFile = new File(srcDir, "config.xml");
+        Files.write(configFile.toPath(), "temp".getBytes());
+
+        util.copyFile(configFile, srcDir, targetDir, "server.xml");
+
+        File targetFile = new File(targetDir, "server.xml");
+        assertTrue(targetFile.exists());
+
+        // clean up
+        if (srcDir.exists()) {
+            FileUtils.deleteDirectory(srcDir);
+        }
+        if (targetDir.exists()) {
+            FileUtils.deleteDirectory(targetDir);
+        }
+    }
+
+    @Test
+    public void testDeleteFile() throws Exception {
+        DevUtil util = new DevTestUtil(null, null, null, null, null, false);
+
+        File srcDir = Files.createTempDirectory("src").toFile();
+        File targetDir = Files.createTempDirectory("target").toFile();
+
+        File tempSrcFile = new File(srcDir, "temp.txt");
+        Files.write(tempSrcFile.toPath(), "temp".getBytes());
+        File tempTargetFile = new File(targetDir, "server.xml");
+        Files.write(tempTargetFile.toPath(), "temp".getBytes());
+
+        // verify that files have been properly created
+        assertTrue(tempSrcFile.exists());
+        assertTrue(tempTargetFile.exists());
+
+        util.deleteFile(tempSrcFile, srcDir, targetDir, "server.xml");
+
+        // verify that the target file has been deleted
+        assertFalse(tempTargetFile.exists());
+
+        // clean up
+        if (srcDir.exists()) {
+            FileUtils.deleteDirectory(srcDir);
+        }
+        if (targetDir.exists()) {
+            FileUtils.deleteDirectory(targetDir);
+        }
+    }
+
+    @Test
+    public void testCleanTargetDir() throws Exception {
+        DevUtil util = new DevTestUtil(null, null, null, null, null, false);
+
+        File outputDirectory = Files.createTempDirectory("outputDirectory").toFile();
+        File tempClass = new File(outputDirectory, "temp.class");
+        Files.write(tempClass.toPath(), "temp".getBytes());
+
+        assertTrue(tempClass.exists());
+
+        util.cleanTargetDir(outputDirectory);
+
+        assertFalse(tempClass.exists());
+
```

Comment:
Is the deletion of an empty outputDirectory expected from the cleanTargetDir method?  If so, the method javadoc could mention that, and assert that it has been deleted here.